### ISSUE TITLE
Google Analytics 4 support

### ISSUE THREAD

**Issue by 8ctopus** [protonemedia/laravel-analytics-event-tracking#16]
Hello!

I'm looking at different options to support server side analytics for Google Analytics 4 and landed on your package. In the readme you say that you support Google Analytics 4 but looking at the project's dependencies you are relying on https://github.com/theiconic/php-ga-measurement-protocol which [doesn't support GA4](https://github.com/theiconic/php-ga-measurement-protocol/issues/94) so I'm a bit confused. Could you please share some light?

**Comment by jussipvainionpaa** [comment]
As Universal Analytics is reaching it's EOL it would be essential to support GA4. Any update on this?

**Comment by jeffreyvanhees** [comment]
@pascalbaljet Any update on this? My custom events are not received by GA4 while they did on Universal Analytics.

**Comment by 8ctopus** [comment]
I'm collaborating on this project which uses measurement protocol for ga4 https://github.com/aawnu/php-ga4/ but there's no laravel implementation.

**Comment by superdejooo** [comment]
Hey guys! Any updates on this? Thanks!

**Comment by LukeTowers** [comment]
Looks like a fork of a fork supports GA4: https://github.com/accexs/laravel-ga4-event-tracking

**Comment by LukeTowers** [comment]
...and now I have forked that fork of a fork: https://github.com/devproca/laravel-ga4-event-tracking. 

@superdejooo @8ctopus @jeffreyvanhees @jussipvainionpaa feel free to try out my fork and let me know if it works for you.

**Comment by 8ctopus** [comment]
@LukeTowers I went for a completely new implementation replicating the gtag.js behavior in php. Why? Because measurement protocol is way too limited in what it can achieve. https://github.com/8ctopus/php-gtag

**Comment by LukeTowers** [comment]
@8ctopus cool! Is there any documentation for the gtag.js API?

**Comment by 8ctopus** [comment]
@LukeTowers No, there is no official documentation from Google that I know of.
